### PR TITLE
Formatting CI prints diff on failure.

### PR DESCRIPTION
**Pull request by 1uc**
As a not very active contributor, it would be convenient if CI told me where I forgot to add that one `" "`. This can be faster than finding out which version of `clang-format` libsonata uses, installing it and then formatting the code. Naturally, in slightly more complex cases one simply needs to install `clang-format`. This mainly decreases the hurdle to submit tiny fixups of typos and the like.

If it doesn't interfere with other peoples workflow, I suggest printing the git diff.

### CODE REVIEW

**Comment by 1uc** [comment]
Currently, the PR contains a commit which intentionally breaks formatting.

**Review by matz-e — APPROVED** [review]

File: src/edge_index.cpp [hunk 1]
```diff
@@ -13,8 +13,8 @@
 
 #include <cstdint>
 #include <set>
-#include <unordered_map>
 #include <vector>
+#include <unordered_map>
```
Comment (matz-e) [inline src/edge_index.cpp:17]:
```suggestion
#include <unordered_map>
#include <vector>
```

**Review by matz-e — DISMISSED** [review]
I like it, but it needs fixes for the CI to pass.

**Comment by mgeplf** [comment]
good call

**Review by mgeplf — APPROVED** [review]

**Pull request merged**